crawler/lookingglass_crawler/resilient_proxy_middleware.py

```
@@ -0,0 +1,82 @@
+                self.enabled = True
+                print(f"✓ Loaded {len(self.proxy_list)} proxies")
+            else:
+                print("ℹ️  No proxies found, disabled")
+        except (FileNotFoundError, IOError) as e:
+            print(f"ℹ️  Could not load proxy file: {e}")
+    
+    @classmethod
+    def from_crawler(cls, crawler):
+        return cls(crawler.settings)
+    
+    def process_request(self, request, spider):
```

Ruff (0.11.9)

Missing return type annotation for public function `process_request`

Add return type annotation: `None`

(ANN201)